DXRando/DeusEx/Classes/MoverToggleTrigger.uc

```diff
@@ -0,0 +1,78 @@
+class MoverToggleTrigger extends FlagToggleTrigger;
+
+var byte keyFrame1;
+var byte keyFrame2;
+var float duration;
+
+var bool moversMoved;
+
+static function MoverToggleTrigger CreateMTT(
+    Actor a,
+    name flagName,
+    name tag,
+    byte keyFrame1,
+    byte keyFrame2,
+    float duration,
+    optional int flagExpiration
+) {
+    if (a == None) return None;
+    return _CreateMTT(a.Spawn(class'MoverToggleTrigger',, tag), flagName, tag, keyFrame1, keyFrame2, duration, flagExpiration);
+}
+
+static function MoverToggleTrigger _CreateMTT(
```

Comment:
maybe a better name for this function is `InitMTT`, and it doesn't really need to be a static function?